sensitive-data/README.md

```diff
@@ -0,0 +1,18 @@
+# Bundle with sensitive data
+
+This bundle demonstrates how porter works with bundle that contains sensitive
+data. 
+
+It requires users to set up a secret store in their porter configuration
+file in order to work with this bundle.
+
+## Try it out 
+
+Follow the steps in the [Change default secret plugin](https://release-v1.porter.sh/end-users/configuration/#change-the-default-secrets-plugin) documentation
+to setup [filesystem](https://release-v1.porter.sh/plugins/filesystem/) plugin.
+After setting up the secret plugin, run to install the bundle:
+```
+porter install --reference ghcr.io/getporter/examples/porter-sensitive-data:v0.1.0
```

Comment:
I think the reference below doesn't match where the bundle will end up being published?

```suggestion
porter install --reference ghcr.io/getporter/examples/sensitive-data:v0.1.0
```

Reply:
When I tried to run the install command without any additional flags the installation failed because one of the parameters is required and does not have a default. Can you update the instructions to include the full command including all required arguments?

```
$ porter install
Building bundle ===>
Defaulting the type of output name to file
Copying porter runtime ===>
Copying mixins ===>
Copying mixin exec ===>

Generating Dockerfile =======>

Writing Dockerfile =======>
Building invocation image
Checking database schema
Selected configured plugin	{"plugin-key": "mongodb"}
Connecting to plugin	{"plugin-command": "/Users/carolynvs/.porter/porter plugin run storage.porter.mongodb"}
starting plugin
plugin started
waiting for RPC address
using plugin
plugin address
Initializing installation collection indices
Initializing parameter collection indices
Initializing credentials collection indices
parameter "password" is required
received EOF, stopping recv loop
plugin process exited
plugin exited
```